solutions/quickstart/version.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_version = ">= 1.9.0"
+
+  # Lock DA into an exact provider version - renovate automation will keep it updated
+  required_providers {
+    ibm = {
+      source  = "IBM-Cloud/ibm"
+      version = "1.83.3"
+    }
+    elasticsearch = {
+      source  = "phillbaker/elasticsearch"
+      version = ">= 2.0.7"
```

Comment:
DAs should lock into exact provider version however I dont think you need this provider as you dont need to create an index

Reply:
Right, this was only added for index creation. I have removed it now.